source/common/config/utility.cc

```diff
@@ -159,6 +159,16 @@ std::chrono::milliseconds Utility::apiConfigSourceRefreshDelay(
       DurationUtil::durationToMilliseconds(api_config_source.refresh_delay()));
 }
 
+std::chrono::milliseconds Utility::apiConfigSourceRequestTimeout(
+    const envoy::api::v2::core::ApiConfigSource& api_config_source) {
+  if (!api_config_source.has_request_timeout()) {
```

Comment:
Sorry forgot to comment on this before but I think you can shorten this to 1 line with PROTOBUF_GET_OPTIONAL_MS(...) (search code)